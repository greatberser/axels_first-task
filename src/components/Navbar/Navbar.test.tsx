import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

describe('Navbar', () => {
  it('renders without crashing', () => {
    const { asFragment } = render(<div>Navbar Component</div>);
    expect(asFragment()).toMatchSnapshot();
  });

  it('contains navigation links', () => {
    render(
      <nav>
        <a href="/">Home</a>
        <a href="/form">Form</a>
      </nav>
    );
    expect(screen.getByText(/home/i)).toBeInTheDocument();
    expect(screen.getByText(/form/i)).toBeInTheDocument();
  });
});
