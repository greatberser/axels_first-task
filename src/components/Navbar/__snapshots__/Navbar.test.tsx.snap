// Jest Snapshot v1, https://jestjs.io/docs/snapshot-testing

exports[`Navbar renders without crashing 1`] = `
<DocumentFragment>
  <div>
    Navbar Component
  </div>
</DocumentFragment>
`;
